upcase_booklist_app/data/users.py

```diff
@@ -0,0 +1,64 @@
+
+
+users = [
+    {
+        'user_id': 1,
+        'user_name': 'Tim McGeorge',
```

Comment:
Since this is already part of the `user` data model, you don't really need to preface the attributes with `user_`. `name`, `biography`, etc is sufficient.